save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/CloneRepositoryController.kt

```diff
@@ -75,6 +84,7 @@ class CloneRepositoryController(
         val projectExecution = executionRequestForStandardSuites.project
         val project = projectService.getProjectByNameAndOwner(projectExecution.name, projectExecution.owner)
         project?.let { proj ->
+            saveExecution(proj, ExecutionType.STANDARD)
```

Comment:
You can discard lambda's argument completely and just use `project`, which is smart-casted to non-nullable type inside this scope. I think it improves readability compared to introducing a new name for the same variable. Or just use `it`, likein the first method.